### PR TITLE
v0.1.0 fpfilter.pl

### PR DESCRIPTION
**Pull request by jasonwalker80**
Download fpfilter v0.1.0 from github instead of including in docker-cle repo.

The new version adds support for bgzip VCF.

### CODE REVIEW

**Review by susannasiebert — APPROVED** [review]
+1

File: Dockerfile [hunk 1]
```diff
@@ -147,8 +147,13 @@ RUN git clone https://github.com/genome/bam-readcount.git /tmp/bam-readcount-0.7
 ##########
 #fpfilter#
 ##########
+WORKDIR /opt
+wget --no-check-certificate https://raw.githubusercontent.com/genome/fpfilter-tool/v0.1.0/fpfilter.pl
```
Comment (tmooney) [inline Dockerfile]: This needs to be preceded by `RUN`

File: Dockerfile [hunk 2]
```diff
@@ -147,8 +147,13 @@ RUN git clone https://github.com/genome/bam-readcount.git /tmp/bam-readcount-0.7
...
 COPY fpfilter.pl /usr/bin/fpfilter.pl
```
Comment (tmooney) [inline Dockerfile]: Since the file is no longer in the repo, this will need to be replaced by a `cp` or `mv`.

File: Dockerfile [hunk 3]
```diff
@@ -147,8 +147,13 @@ RUN git clone https://github.com/genome/bam-readcount.git /tmp/bam-readcount-0.7
...
 
+WORKDIR /
+rm /opt/fpfilter.pl
```
Comment (tmooney) [inline Dockerfile]: Another `RUN` here.

(For maximum efficiency, the `wget`, move/copy, and `rm` could all be in one `RUN` to avoid extra layers.)

**Review by tmooney — CHANGES_REQUESTED** [review]

**Review by tmooney — APPROVED** [review]

**Pull request merged**